interactions:
- request:
    body: null
    headers:
      Accept:
      - '*/*'
      Accept-Encoding:
      - gzip, deflate
      Connection:
      - keep-alive
      User-Agent:
      - python-requests/2.28.1
    method: GET
    uri: https://zenodo.org/api/records/68358988
  response:
    body:
      string: '{"status": 404, "message": "PID does not exist."}'
    headers:
      Access-Control-Allow-Origin:
      - '*'
      Access-Control-Expose-Headers:
      - Content-Type, ETag, Link, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset
      Content-Length:
      - '49'
      Content-Type:
      - application/json
      Date:
      - Sun, 18 Sep 2022 12:38:17 GMT
      Referrer-Policy:
      - strict-origin-when-cross-origin
      Retry-After:
      - '60'
      Server:
      - nginx
      Strict-Transport-Security:
      - max-age=0
      X-Content-Type-Options:
      - nosniff
      X-Frame-Options:
      - sameorigin
      X-RateLimit-Limit:
      - '60'
      X-RateLimit-Remaining:
      - '52'
      X-RateLimit-Reset:
      - '1663504758'
      X-XSS-Protection:
      - 1; mode=block
    status:
      code: 404
      message: NOT FOUND
version: 1
